pom.xml

```diff
@@ -53,13 +53,13 @@
     <version>1.0.0</version>
 
     <properties>
-        <maven.compiler.source>8</maven.compiler.source>
-        <maven.compiler.target>8</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <grpc.version>1.43.1</grpc.version>
+        <grpc.version>1.45.0</grpc.version>
         <protobuf.version>3.19.4</protobuf.version>
-        <payara.version>5.2022.1</payara.version>
-        <payara.internal.version>5.2020.5</payara.internal.version>
+        <payara.version>6.2022.1.Alpha3-SNAPSHOT</payara.version>
```

Comment:
Should we need to consider to create a new release of the internal dependencies? I think for now you are using the SNAPSHOT but we need to upgrade in some moment with the final release

Reply:
Yes ideally we would be using a final version.
It only appears to be pulling in the bom and no other internal dependencies, so we can presumably just set this to 6.2022.1.Alpha2?